styles/dialog/fight.scss

```diff
@@ -19,13 +19,71 @@
         }
     }
 
+    .details {
+        display: grid;
+        grid-template-columns: repeat(4, 1fr);
+        border: 2px groove #aaaaaa;
+        justify-items: stretch;
+        column-gap: 2px;
+        row-gap: 2px;
+        .details-key{
+            display: flex;
+            align-items: center;
+            background: rgba(#000, .03);
+            border: 2px groove #aaaaaa;
+            padding: 1px;
+        }
+        .details-value{
+            display: flex;
+            align-items: center;
+            flex-direction: row-reverse;
```

Comment:
Why reverse?

Reply:
So that items "stick" to the right and not left by default.